crypto/Cargo.toml

```diff
@@ -31,6 +32,10 @@ path = "../keystore"
 version = "0.3"
 path = "../mls-provider"
 
+[dependencies.core-crypto-attributes]
```

Comment:
Since it's only used in test configuration, maybe move it to `dev-dependencies`?